Artifacts/MsSql/Structure/Admin/0130-Add-ProfileDefinition-to-Profile.sql

```diff
@@ -0,0 +1,7 @@
+-- SPDX-License-Identifier: Apache-2.0
+-- Licensed to the Ed-Fi Alliance under one or more agreements.
+-- The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+-- See the LICENSE and NOTICES files in the project root for more information.
+
+ALTER TABLE [dbo].[Profiles]
+    ADD [ProfileDefinition] [nvarchar](max) NULL;
```

Comment:
Can you use  the XML data type to   store the XML data instead of [nvarchar]  like ADD [ProfileDefinition] XML NULL;

Reply:
I'm not sure I agree with this recommendation. We are not using, and do not plan to use any XML-specific features of the database engines (SQL Server or Postgres). My suggestion would be to leave this as an `nvarchar` column. @vimayya? Thoughts?

Reply:
we can reduce stored data size if we use the xml type (at least in MSSQL, not really sure in PostgreSql)

Reply:
I am on the fence on this one. Does using the xml complicate C# implementation due to differences in database servers? We could stay with nvarchar and investigate XML type in a separate ticket before the release. Change after the release complicates migration/upgrade scenarios.